### PR TITLE
Merge pull request #287 - partners no longer go on waitlist.

### PR DESCRIPTION
**Pull request by 18Aayushk**
Workshop cap

🎟️ **Ticket(s):**
Closes #
Partners are getting a Waitlist email when the event is full

👷 **Changes:**
A brief summary of what changes were introduced.
Partners bypass "event is full" check on backend.

💭 **Notes:**
Any additional things to take into consideration.

**Wait! Before you merge, have you checked the following:**
- [ ] Serverless tests are passing (Check travis build logs, CI is currently broken)
- [ ] PR is has approving review(s)

Screen Shot 2023-10-03 at 13 31 57

### CODE REVIEW

File: services/registrations/handler.js [hunk 1]
```diff
@@ -217,13 +226,13 @@ export async function sendEmail(user, existingEvent, registrationStatus, id) {
     let tempId = "d-11d4bfcbebdf42b686f5e7d0977aa952";
     let tempCalendarId = "d-b517e4c407e4421a8886140caceba551";
 
-    if (registrationStatus === "cancelled") {
+    if (registrationStatus == "cancelled") {
```
Comment (ddennis924) [inline services/registrations/handler.js:229]: is there a reason why we changed these from strict equals to ==?

Comment (18Aayushk) [reply services/registrations/handler.js:229]: No reason. can change it back, I'm not sure why my github like defaulted into recommending all those === to == changes. Should I change it back?

**Review by ddennis924 — COMMENTED** [review]
lets get this merged in @18Aayushk please iomplement the above changes and fix merge conflicts, if you need help with this, let me know or come to office hours

**Comment by voctory** [comment]
Closing stale PRs in `ubc-biztech` org

**Pull request closed without merging**